### PR TITLE
feat: medical program setting added to medical programs

### PR DESCRIPTION
**Pull request by SergeyTatarenko**

### CODE REVIEW

File: __schema__/medicalPrograms.graphql [hunk 1]
```diff
@@ -76,6 +76,10 @@ type MedicalProgram implements Node {
   medicationDispenseAllowed: Boolean!
   "Medication dispense allowed status description"
   medicationDispenseAllowedText: String
+  "Medical program settings"
+  mediaclProgramSettings: MediaclProgramSettings!
+  "Medical program settings status description"
+  mediaclProgramSettingsText: String
```
Comment (kiurchv) [inline __schema__/medicalPrograms.graphql:82]:
```suggestion
  medicalProgramSettings: MediaclProgramSettings!
  "Medical program settings status description"
  medicalProgramSettingsText: String
```

File: __schema__/medicalPrograms.graphql [hunk 2]
```diff
@@ -141,3 +145,21 @@ type DeactivateMedicalProgramPayload {
   "Deactivated `MedicalProgram`."
   medicalProgram: MedicalProgram
 }
+
+"""
+Medical program settings.
+"""
+type MediaclProgramSettings {
```
Comment (kiurchv) [inline __schema__/medicalPrograms.graphql:152]:
```suggestion
type MedicalProgramSettings {
```

Comment (SergeyTatarenko) [reply __schema__/medicalPrograms.graphql]: fixed in last last commit

File: __schema__/medicalPrograms.graphql (hunk 1 above)
Comment (SergeyTatarenko) [reply __schema__/medicalPrograms.graphql]: fixed in last last commit

**Comment by SergeyTatarenko** [comment]
PR for review only (not merge for now)

**Review by nemaslov — DISMISSED** [review]

**Review by nemaslov — APPROVED** [review]

**Review by imax-iva — APPROVED** [review]

**Pull request closed without merging**